lib/splitclient-rb/cache/filter/flag_set_filter.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'set'
+
+module SplitIoClient
+  module Cache
+    module Filter
+      class FlagSetsFilter
+        def initialize(flag_sets = [])
+          @flag_sets = Set.new(flag_sets)
+          @should_filter = @flag_sets.any?
+        end
+
+        def flag_set_exist?(flag_set)
+          if not @should_filter
+            return true
+          end
```

Comment:
you can replace this `if` for this line: ` return true unless @should_filter`

Reply:
will do